bundles/org.openhab.ui/web/src/res/img/openhab-logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 170 37">
+	<style>
+		@media (prefers-color-scheme: dark) {
```

Comment:
This isn't quite right because the user can override the preferred color scheme in the help & about page.
When the dark mode is active though, the root html element will have the `theme-dark` CSS class applied.

Reply:
Actually, I’d skip that and stick to the „coloured“ version. I think it looks much better than the white in any case, including the favicon.
![E92A9593-0BA1-4AA4-904B-ECF9F54A059C](https://user-images.githubusercontent.com/2242688/116758536-8c997d00-aa10-11eb-8ec4-78cc32b7abd1.jpeg)

Reply:
I'm okay with either. Which would you prefer @ghys?

Reply:
I made a white only image deliberately because IMO the contrast ratio of the gray circle/"HAB" against a dark the background is poor, especially compared to the orange parts which stand out as a result.
As for the favicon, in the png version the logo is inside a white circle.

I don't think it looks horrible either way but I'd stick to the white version in dark mode.

Reply:
I'm not super familiar with this web system, any tips on the right css for the override on the svgs icons?

Reply:
I don't think there's a straightforward solution that makes it possible to use inline CSS styling like the current `prefers-color-scheme: dark`.
I think for this to work the SVG has to be loaded inline, which would require using another webpack loader.
So, it's probably best to stick to two images.